doc/changelog/02-specification-language/19519-coercion-field-cleanup.rst

```diff
@@ -0,0 +1,8 @@
+- **Changed:**
+  ``:>`` now always declares coercions. The previous behavior,
```

Comment:
```suggestion
  ``:>`` now always declares coercions when used with :cmd:`Record` and :cmd:`Class`. The previous behavior,
```

Reply:
It's also true in other contexts, it's really always.

Reply:
I don't know, it may be confusing too.

Reply:
maybe 
```suggestion
  ``:>`` in :token:`of_type_inst` now always declares coercions. The previous behavior,
```
https://coq.inria.fr/doc/master/refman/language/core/records.html#grammar-token-of_type_inst

Reply:
Great, let's do that.